web/models.py

```diff
@@ -42,7 +45,7 @@ class Income(models.Model):
     text = models.CharField(max_length=255)
     date = models.DateTimeField()
     amount = models.BigIntegerField()
-    user = models.ForeignKey(User)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
 
```

Comment:
also fixed some bugs with ForeignKey it showed error that on_delete attribute has to be defiend